src/sentry/middleware/ratelimit.py

```diff
@@ -48,7 +57,7 @@ def process_view(self, request: Request, view_func, view_args, view_kwargs) -> R
                 return
 
             request.rate_limit_metadata = above_rate_limit_check(
-                request.rate_limit_key, rate_limit, request.rate_limit_uid
+                data["rate_limit_key"], rate_limit, data["rate_limit_uid"]
```

Comment:
is there a reason `rate_limit_metadata` still needs to be attached to the `request`?

Reply:
Pretty much just because I can't figure out how else to get it into the access log.